lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3814,11 +3848,15 @@ namespace Js
                     {
                         threadContext->GetDebugManager()->GetDebuggingFlags()->SetIsBuiltInWrapperPresent(false);
                     }
+#ifndef DISABLE_SEH
                     __try
+#endif
                     {
                         aReturn = JavascriptFunction::CallFunction<true>(function, origEntryPoint, args);
                     }
+#ifndef DISABLE_SEH
                     __finally
+#endif
                     {
                         threadContext->GetDebugManager()->GetDebuggingFlags()->SetIsBuiltInWrapperPresent(isOrigWrapperPresent);
```

Comment:
@obastemur The `TryFinally` function I mentioned has nothing to do with `libunwind / seh`, and why "subject to removal"?